test/unit/core/store/middleware.ts

```diff
@@ -167,14 +154,70 @@ suite('Middleware', ({ expect, spy, stub }) => {
       const action = { type: Actions.RECEIVE_PRODUCTS, error: true };
       const undoAction = { a: 'b' };
       const next = spy();
-      stub(ActionCreators, 'undo').returns(undoAction);
+      stub(ReduxActionCreators, 'undo').returns(undoAction);
 
       Middleware.errorHandler(<any>{})(null)(next)(action);
 
       expect(next).to.be.calledWith(undoAction);
     });
   });
 
+  describe('saveStateAnalyzer()', () => {
+    it('should pass through other actions', () => {
+      const next = spy();
+      const getState = spy();
```

Comment:
we aren't making any expectations about `next()` or `getState()` so why do they need to be `spy()`s? and why do we need to save them as `const`s at all?

Reply:
Next is being used.